app/index.js

```diff
@@ -82,23 +79,23 @@ exports = module.exports = generators.Base.extend({
     this._makeDestDir('my_inc');
   },
   copyFiles: function(){
-    this._copyToDest('._gitignore', '.gitignore');
-    this._copyToDestWithTemplate('_configure.ac', 'configure.ac', this.answers);
-    this._copyToDest('_Makefile.am', 'Makefile.am');
-    this._copyToDestWithTemplate('_AUTHORS', 'AUTHORS', this.answers);
-    this._copyToDestWithTemplate('_LICENCE.md', 'LICENCE.md', this.answers);
-    this._copyToDestWithTemplate('_README.md', 'README.md', this.answers);
+    this.copy('_gitignore', '.gitignore');
+    this.template('_configure.ac', 'configure.ac', this.answers);
+    this.copy('_Makefile.am', 'Makefile.am');
+    this.template('_AUTHORS', 'AUTHORS', this.answers);
+    this.template('_LICENCE.md', 'LICENCE.md', this.answers);
+    this.template('_README.md', 'README.md', this.answers);
```

Comment:
Seems like Yeoman updated their API so that the this.copy/template uses this.fs.copy/template